### PR TITLE
Fix PiCT reported bugs + build enhancements (12).

### PR DESCRIPTION
**Pull request by swoocn**
- Improve performance of search queries within user's viewpoint; bumped max number of displayed seller markers to 50.
- Resolve Search Center default not correctly set to IP location.
- Resolve 'Seller Registration' bug: phone number input cursor behavior.
- Introduce French language.
- Introduce Simplified Chinese language.
- Update App Version to 'v1.3.0'.

**[see commits]**

_This PR is subject to change if additional fixes + enhancements are introduced before approval._

### CODE REVIEW

**Comment by swoocn** [comment]
Hi @tate1650 - Could you please keep this PR open and refrain from merging until I give the word? 
Thanks!

**Comment by swoocn** [comment]
Hi @tate1650 - I added some post' comments to [https://github.com/map-of-pi/map-of-pi-docker/pull/31](https://github.com/map-of-pi/map-of-pi-docker/pull/31) regarding failure with our unit tests.  Could you please address accordingly?  Thx!

**Comment by tate1650** [comment]
Alright, so just double-checking, I'm still waiting for the thumbs-up on this one, right?

File: backend/test/jest.setup.ts [hunk 1]
```diff
@@ -29,6 +29,8 @@ beforeAll(async () => {
     // Load the mock data into Map of PI DB collections
     await User.insertMany(mockData.users);
     await UserSettings.insertMany(mockData.userSettings);
+    // Ensure indexes are created for the Seller model before running tests
+    await Seller.createIndexes();
```
Comment (swoocn) [inline backend/test/jest.setup.ts:33]: Creating indexes for the Seller model appears to have resolved the inconsistent unit test failures, effectively addressing the issue mentioned in PR #31 @ https://github.com/map-of-pi/map-of-pi-docker/pull/31#issuecomment-2492570074.

**Comment by swoocn** [comment]
> Alright, so just double-checking, I'm still waiting for the thumbs-up on this one, right?

That is correct! I will let you know soon!

**Comment by swoocn** [comment]
> > Alright, so just double-checking, I'm still waiting for the thumbs-up on this one, right?
> 
> That is correct! I will let you know soon!

Hi @tate1650 - Please review and approve the changes when you can so we can proceed with the deployment propagation.  Thx!

**Review by tate1650 — APPROVED** [review]
Looks good. I'll go ahead and deploy this on staging, but I won't be able to get it out to production until Monday if that's alright.

**Comment by tate1650** [comment]
Update: changes are currently on staging

**Comment by swoocn** [comment]
No problem @tate1650; I will keep you posted with the propagation.  Thanks again!

**Comment by swoocn** [comment]
> No problem @tate1650; I will keep you posted with the propagation. Thanks again!

Hi @tate1650 - Staging validation has been completed successfully.  Please proceed to deploy to prod at the earliest and let me know.  Thx!

**Pull request merged**